docs/admin/addons.rst

```diff
@@ -32,8 +32,8 @@ components.
 
 .. _addon-weblate.cleanup.generic:
 
-Cleanup translation files
--------------------------
+Update PO files to match POT (msgmerge)
+---------------------------------------
 
 Update all translation files to match the monolingual base file. For most file
 formats, this means removing stale translation keys no longer present in the
```

Comment:
Can't remember if there is a newer word for "translation keys".

Reply:
We use keys elsewhere as well (recently introduced "Nearby keys"), so that's probably okay

Reply:
@nijel How is "nearby keys" different from "nearby strings". I have used it and can't really tell…

Reply:
"nearby keys" orders strings by keys and lists nearby ones, "nearby strings" uses ordering as is in the translation file